










try {
  print("foo: {foo()}")
} catch (err) {
  print("error: {err}")
} finally {
  print("finally handlers")
}
